misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -161,8 +161,8 @@ public void test(String fileName, String path) throws IOException {
                 "annot_decl_source_02.bal", "class_def_source_11.bal", "object_type_def_source_26.bal",
                 "ambiguity_source_32.bal", "service_object_constructor_source_01.bal",
                 "service_object_constructor_source_02.bal", "object_constructor_source_08.bal",
-                "annotations_source_01.bal", "do_stmt_source_08.bal", "func_params_source_10.bal",
-                "func_params_source_11.bal", "func_params_source_13.bal");
+                "annotations_source_01.bal", "do_stmt_source_08.bal", "func_params_source_14.bal",
+                "func_params_source_15.bal", "func_params_source_13.bal", "func_params_source_11.bal");
```

Comment:
we need to run the formatter for negative tests as well. Shall we not not skip  func_params_source_13., func_params_source_14 and func_params_source_15

func_params_source_11 is fine as the formatter currently does not loop the invalid tokens in the minutiae.

Reply:
Change as mentioned in https://github.com/ballerina-platform/ballerina-lang/pull/27662#discussion_r553565834